dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_tool.jsp

```diff
@@ -279,7 +311,7 @@
                     <button  dojoType="dijit.form.Button" onClick="showBundleUpload();" iconClass="uploadIcon">
                         <%= LanguageUtil.get(pageContext, "publisher_upload") %>
                     </button>
-                    <button dojoType="dijit.form.Button" onClick="deleteAudits();" id="deleteAuditsBtn" iconClass="deleteIcon" class="dijitButtonDanger">
+                    <button dojoType="dijit.form.Button" onClick="deleteBundlesOptions();" id="deleteAuditsBtn" iconClass="actionIcon" class="dijitButtonDanger">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)